### PR TITLE
Add telegram disable_web_page_preview (telegram_bot.markdown)

### PR DESCRIPTION
**Pull request by ildar170975**
## Proposed change

Adds ability to specify disable_web_page_preview to telegram.notify

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/75898
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Would be good to merge these changes into https://github.com/home-assistant/home-assistant.io/pull/23572.

File: source/_posts/2022-07-27-leviton-partner.markdown [hunk 1]
```diff
@@ -31,4 +31,4 @@ Leviton Decora Smart Z-Wave Devices include their DZ6HD dimmer, DZ15S switch, ZW
 
 If you would like to learn more about the Leviton products and integration check out their [integration page](/integrations/leviton)! Leviton will also show up in the integrations list in Home Assistant version 2022.8!
 
-If you missed the announcement of the Works with Home Assistant partner program, make sure to check out the [blog post](/partner-program).
+If you missed the announcement of the Works with Home Assistant partner program, make sure to check out the [blog post](/blog/2022/07/12/partner-program/).
```
Comment (MartinHjelmare) [inline source/_posts/2022-07-27-leviton-partner.markdown]: Please don't include this change in this PR.

Comment (frenck) [reply source/_posts/2022-07-27-leviton-partner.markdown]: Rebased the PR

**Review by frenck — APPROVED** [review]
Thanks, @ildar170975 👍

**Comment by ildar170975** [comment]
Thanks a lot for trusting)

**Pull request merged**